### PR TITLE
feat: multi region prod deploy

### PR DESCRIPTION
**Pull request by francardoso93**

### CODE REVIEW

File: .github/workflows/deploy-main-us-east-2.yml [hunk 1]
```diff
@@ -52,5 +52,5 @@ jobs: # We need two separate builds as each is going to a different ECR repo
     secrets:
       aws_account_id: ${{ secrets.AWS_ACCOUNT_ID }}
       aws_role_id: ${{ secrets.AWS_ROLE_ID }}
-      aws_region: ${{ secrets.AWS_REGION }}
-      aws_ecr_repository: ${{ secrets.AWS_ECR_REPOSITORY }}
+      aws_region: ${{ secrets.AWS_REGION_PROD_0 }}
```
Comment (alanshaw) [inline .github/workflows/deploy-main-us-east-2.yml:55]:
```suggestion
      aws_region: us-east-2
```

File: .github/workflows/deploy-main-us-west-2.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      id-token: write
+    needs: build-prod
+    with:
+      deploy_to_environment: prod
+      lambda: side-indexer
+      docker_image_version: ${{ needs.build-prod.outputs.build_date }}
+    secrets:
+      aws_account_id: ${{ secrets.AWS_ACCOUNT_ID }}
+      aws_role_id: ${{ secrets.AWS_ROLE_ID }}
+      aws_region: ${{ secrets.AWS_REGION_PROD_1 }}
```
Comment (alanshaw) [inline .github/workflows/deploy-main-us-west-2.yml:31]:
```suggestion
      aws_region: us-west-2
```

**Review by alanshaw — APPROVED** [review]

File: .github/workflows/deploy-main-us-east-2.yml (hunk 1 above)
Comment (francardoso93) [reply .github/workflows/deploy-main-us-east-2.yml]: Done

File: .github/workflows/deploy-main-us-west-2.yml (hunk 2 above)
Comment (francardoso93) [reply .github/workflows/deploy-main-us-west-2.yml]: done

**Pull request merged**